### DISCUSSION TITLE
Feature request Dashboard

### DISCUSSION THREAD

**Discussion by prawnhitter** [solectrus/solectrus#1867] [Ideas]
First of all i would like to say that it is an awesome tool. Also with the import feature. That worked ery good.

I would like to have three suggestions:

- [ ] The Main Dashboard is showing on the right side only the gerated power. There would be a dropdown nice what the user wants to see. For Example i would like to see more the "Autarkie" Level through the day.
- [ ] Regarding the actual topics with Senec and their porblems, i would like to have dashboard with an overview of Hausverbrauch, Batterieladestand, Erzeugung und Bezug/Zukauf. To see if everything is working correctly.
- [ ] As i imported all mein Data from the csv Data, the weekly, monthly and yearly dashboards showed them correct, but the daily and now Dashboard were only showing the live collected data. Maybe just an grafical error, but even after i restarted all containers these blind spots occured.

But still thank you for your great effort here.

**Comment by ledermann** [comment]
Thanks a lot! Regarding your suggestions:

1. There are lots of charts available. Just click on the elements (bar segments or circles) on the left
2. I don't understand, these values are already displayed. Please give more details.
3. Maybe this is a caching issue. After the import, you should delete the REDIS cache (as described [in the README](https://github.com/solectrus/senec-importer)). But note: The "Now" tab displays only the live data, not the imported data.

**Reply by prawnhitter** [reply]
@ledermann  Ich würde einfach mal ins Deutsche wechseln :) das ist grad etwas einfacher.

Bzgl. Punkt 1:
Ich habe gesehen das man über den Umweg "Top10" eine andere Statistik auswählen kann und diese dann ins "Now" transferiere. Die Überlegung ging mehr dahin das jeder User sich das Dashboard selber customizable macht. 
Ich persönlich präferiere eben wenn ich die URL Aufrufe direkt den Batterieverlauf zu sehen.

Zu Punkt 2:
Ich habe vergessen zu schreiben das diese Daten in einer Übersicht dargestellt werden.

![image](https://user-images.githubusercontent.com/50422850/228780568-acf06797-a9d7-4c37-933e-d951c027fcce.png)

Also Erzeugung (kw) als Fläche, Batteriestand (%) als Linie, Hausverbrauch (kw) als Fläche und Zukauf/Bezug (kw) auch wieder als Line

Zu das man sehen kann ob der Wechselrichter sauber läuft etc.

Ich hoffe dies war nun etwas verständlicher :)

**Reply by ledermann** [reply]
Zu 1) Ja, auch die Top10 ist verlinkt. Aber der einfachste Weg, das Diagramm auf der **rechten** Seite zu wechseln, ist ein Klick auf eines der Elemente auf der **linken** Seite: Ein Klick links auf "Hausverbrauch" z.B. wechselt auf das zugehörige Diagramm. Auch die Badges (Kreise) für Autarkie und Verbrauch sind verlinkt. Zukünftig erreichbar man dadurch auch Diagramme für Ladestand (#1734) und Temperatur (#1054).
Die Darstellung von SOLECTRUS ist derzeitig **nicht** customizable. Ich schätze, das wird auch so bleiben. Da jede Darstellung aber eine eigene URL hat, kannst Du den Link für eine bevorzugte Darstellung direkt aufrufen.

Zu 2) Du zielst vermutlich auf eine gleichzeitige Darstellung unterschiedlicher Werte in einem einzigen Diagramm ab. Mir persönlich ist das zu wild bzw. zu technisch, das strebe ich eher nicht an. Sorry :-)

**Reply by prawnhitter** [reply]
@ledermann okay, kein Problem :-) aber ja das war das Ziel zu Punkt 2 dann muss ich das wohl versuchen selber zu visualisieren über die Datenbank. Trotzdem vielen Dank für die echt gut ausgearbeitete Applikation

**Comment by prawnhitter** [comment]
Topic closed
